src/release/release.yml

```diff
@@ -0,0 +1,29 @@
+# Orb Version 0.0.1
+
+version: 2.1
+description: Wrapper for Artsy's release management tools
+
+executors:
+  scripts:
+    docker:
+      - image: artsy/orb-scripts
+
+jobs:
+  block:
+    executor: scripts
+    parameters:
+      project_id:
+        type: integer
+        description: The primary key of the project object in horizon
+    steps:
+      - run:
+          description:
```

Comment:
@zephraph I think we should add a `description` so that future developers best understand this code.

Reply:
Oh yeah! I'm surprised this doesn't break 😄

Reply:
```suggestion
          description: Check horizon to see if there's an unresolved deployment block
```